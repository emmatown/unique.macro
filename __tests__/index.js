import tester from 'babel-plugin-tester'

tester({
  plugin: require('babel-macros'),
  snapshot: true,
  tests: {
    basic: {
      code: `import unique from 'unique.macro'
      const uniqueClass = unique()`
    },
    'with commonjs': {
      code: `const unique = require('unique.macro')
      const uniqueClass = unique()`
    },
    'custom prefix': {
      code: `import unique from 'unique.macro'
      const uniqueSomething = unique('something-')`
    }
  }
})
