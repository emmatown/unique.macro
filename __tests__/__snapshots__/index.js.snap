// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unique.macro basic 1`] = `
"
const uniqueClass = 'css-1xp6fv5';"
`;

exports[`unique.macro custom prefix 1`] = `
"
const uniqueSomething = 'something-13atynq';"
`;

exports[`unique.macro with commonjs 1`] = `
"
const uniqueClass = 'css-12y3co1';"
`;
